rescompy/rescompy.py

```diff
@@ -279,8 +339,9 @@ def reservoir_outputs(self):
     def rmse(self):
         """The root-mean-square error property.
         Computes the root-mean-square error as a function of time."""
-        return np.sqrt(np.mean(np.square(self.reservoir_outputs -
-                                         self.target_outputs), axis=1))
+        return np.sqrt(np.mean(
+			np.square(self.reservoir_outputs -
+			 self.target_outputs[self.lookback_length:]), axis=1))
     
```

Comment:
what is `lookback_length`?  is it zero usually?

Reply:
lookback_length is the greater of inputs_lookback_length and states_lookback_length i.e., the total lookback_length required to form feature vectors. It is zero whenever the features do not include time-delayed states or time-delayed inputs.